contrib_bots/bots/thesaurus/thesaurus.py

```diff
@@ -21,24 +21,26 @@ def get_clean_response(m, method):
         return response
     elif isinstance(response, list):
         return ', '.join(response)
+    else:
+        return "Not found"
 
 def get_thesaurus_result(original_content):
-    search_keyword = original_content.strip().split(' ', 1)[1]
-    if search_keyword == 'help':
-        help_message = "To use this bot, start messages with either \
-                        @mention-bot synonym (to get the synonyms of a given word) \
-                        or @mention-bot antonym (to get the antonyms of a given word). \
-                        Phrases are not accepted so only use single words \
-                        to search. For example you could search '@mention-bot synonym hello' \
-                        or '@mention-bot antonym goodbye'."
+    help_message = "To use this bot, start messages with either \
+                            @mention-bot synonym (to get the synonyms of a given word) \
+                            or @mention-bot antonym (to get the antonyms of a given word). \
+                            Phrases are not accepted so only use single words \
+                            to search. For example you could search '@mention-bot synonym hello' \
+                            or '@mention-bot antonym goodbye'."
+    if len(original_content.strip().split(' ', 1))<2:
```

Comment:
You have `original_content.strip().split(' ', 1)` twice.  Would it make sense to define it as a separate variable to make the code easier to read?

```
query = original_content.strip().split(' ', 1)
if len(query) < 2:
    return help_message
...
```